code/datums/mutations.dm

```diff
@@ -673,3 +690,45 @@
 				mut_overlay |= V
 				overlays_standing[CM.layer_used] = mut_overlay
 				apply_overlay(CM.layer_used)
+
+/datum/mutation/human/armblade
+
+	name = "Armblade"
+	quality = POSITIVE
+	get_chance = 25
+	lowest_value = 256 * 12
+	text_gain_indication = "<span class='notice'>You can turn your arm into a sword! Sweet!</span>"
+	text_lose_indication = "<span class='notice'>You have normal, non-sword arms again. Damnit.</span>"
+
+/datum/mutation/human/armblade/on_acquiring(mob/living/carbon/human/owner)
+	if(..())	return
+	owner.verbs += /mob/living/carbon/human/verb/armblade
+
+/datum/mutation/human/armblade/on_losing(mob/living/carbon/human/owner)
+	if(..())	return
+	owner.verbs -= /mob/living/carbon/human/verb/armblade
+
+/mob/living/carbon/human/verb/armblade()
+	set name = "Toggle Armblade"
+	set category = "Mutations"
+	set desc = "Reform your arm into a grotesque organic blade"
+	living_weapon(src, /obj/item/weapon/melee/arm_blade)
+
+/mob/living/carbon/human/proc/living_weapon(mob/user, obj/item/weapon_type, right_hand=1)
+	var/obj/item/hand_item = user.get_active_hand()
+	if(hand_item && istype(hand_item, weapon_type))
+		playsound(user, 'sound/effects/blobattack.ogg', 30, 1)
+		user.visible_message("<span class='warning'>With a sickening crunch, [user] reforms their [hand_item.name] into an arm!</span>", "<span class='notice'>You assimilate the [hand_item.name] back into our body.</span>", "<span class='italics>You hear organic matter ripping and tearing!</span>")
```

Comment:
There should be a way to produce a different message and sound, because this message is very changeling-y and you might want some other sort of message and sound.